src/o365/spfx/commands/project/project-rename.ts

```diff
@@ -90,81 +90,101 @@ class SpfxProjectRenameCommand extends BaseProjectCommand {
       }
     }
 
-    const filePaths: string[] = [
-      path.join(this.projectRootPath, 'package.json'),
-      path.join(this.projectRootPath, '.yo-rc.json'),
-      path.join(this.projectRootPath, 'config/package-solution.json'),
-      path.join(this.projectRootPath, 'config/deploy-azure-storage.json'),
-      path.join(this.projectRootPath, 'README.md')
-    ];
+    const replacePackageJsonContent = (filePath: string) => {
+      if (!fs.existsSync(filePath)) {
+        return;
+      }
+      const existingContent = fs.readFileSync(filePath, 'utf-8');
+      const updatedContent = JSON.parse(existingContent);
+      if (updatedContent &&
+        updatedContent.name) {
+        updatedContent.name = args.options.newName;
+      }
+      fs.writeFileSync(filePath, JSON.stringify(updatedContent, null, 2), 'utf-8');
+      if (this.debug) {
+        cmd.log(`Updated ${filePath.split('/').pop()}`);
```

Comment:
Rather than using fixed `/` we should use OS-specific path separator (`path.sep`). In this particular case, to get the file-/directory name from the path, it would be better to use `path.basename` (https://nodejs.org/api/path.html#path_path_basename_path_ext)